src/Backend/Modules/Slideshows/Actions/Edit.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Backend\Modules\Slideshows\Actions;
 
+use Symfony\Component\Finder\Finder;
+use Frontend\Core\Engine\Theme;
```

Comment:
do you use these 2 use statements in this class?
They have been added but I don't see a use in the added lines